ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.17.yaml

```diff
@@ -84,12 +84,18 @@ tests:
     test:
     - ref: fips-check-image-scan
 - as: e2e-operator
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.19"
+  skip_if_only_changed: ^docs/|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
```

Comment:
nit: can we also add deploy/examples, hack, jsonnet ?

Reply:
Updated, but didn't include `hack/` as they might be highly relevant to build and verify